docker/run.sh

```diff
@@ -66,10 +66,31 @@ while [[ "$1" != "" ]]; do
     esac
 done
 
+DOCKER_EXTRA_ARGS=""
+BASE_PATH="$(cd .. && pwd)"
+
+if [ "${ROSDISTRO}" == "humble" ]; then
+    DOCKER_EXTRA_ARGS="${DOCKER_EXTRA_ARGS} -v /home/developer/ws/src/beluga_demo/localization/beluga_demo_amcl3_localization"
+fi
+
+if [ "${ROSDISTRO}" == "jazzy" ]; then
+    EXCLUDE_FOLDERS=("common" 
+                     "integration" 
+                     "misc" 
+                     "localization/beluga_demo_bearing_localization" 
+                     "localization/beluga_demo_fiducial_localization" 
+                     "localization/beluga_demo_lidar_localization"
+    )
```

Comment:
@pvela2017 I'm a bit ambivalent about this, but I can't think of a better solution. CC @glpuga for thoughts.